src/main/java/io/xpring/xrpl/model/XRPCurrencyAmount.java

```diff
@@ -54,9 +48,8 @@ static XRPCurrencyAmount from(CurrencyAmount currencyAmount) throws NumberFormat
     switch (currencyAmount.getAmountCase()) {
       // Mutually exclusive: either drops or issuedCurrency is set in an XRPCurrencyAmount
       case ISSUED_CURRENCY_AMOUNT: {
-        IssuedCurrencyAmount issuedCurrencyAmount = currencyAmount.getIssuedCurrencyAmount();
-        if (issuedCurrencyAmount != null) {
-          XRPIssuedCurrency xrpIssuedCurrency = XRPIssuedCurrency.from(issuedCurrencyAmount);
+        if (currencyAmount.hasIssuedCurrencyAmount()) {
```

Comment:
From what I've noticed digging around in the protobuf code for Java, they're designed to not ever return null... instead they return default empty instances of things and handle null things internally.  I'm still not entirely sure what this means when it comes to the value of primitives such as `int` or `string` ... I know empty strings are returned in the absence of value. 

Anyway, yes these boolean `hasWhateverField` methods are a way of checking whether the field in question has been set.  In this case, I actually just deleted this line because it's been replaced with a switch/case statement and I forgot to delete it.